chess-AI/nn_layout.py

```diff
@@ -23,116 +23,94 @@ class PlayNetwork(nn.Module):
     """
     def __init__(self):
         super(PlayNetwork, self).__init__()
-        self.num_res_blocks = 3
+        self.num_res_blocks = 40
+        self.num_filters = 32
         
         # Takes 119 channels of input comprised of 7 previous states' piece and repetition
         # planes plus the current state input of 21 channels.
-        self.conv_layer = nn.Sequential(nn.Conv2d(in_channels=119,
-                                                  out_channels=256,
+        self.conv_layer = nn.Sequential(nn.Conv2d(in_channels=19,
+                                                  out_channels=self.num_filters,
                                                   kernel_size=3,
                                                   padding=1,
                                                   bias=False),
-                                        nn.BatchNorm2d(256),
-                                        nn.ReLU(inplace=True))
+                                        nn.BatchNorm2d(self.num_filters),
+                                        nn.ReLU())
         
         # Each conv layer in a residual block uses 256 3x3 filters, padding used
         # to keep the channel dimensions constant.
-        self.res_block = nn.Sequential(nn.Conv2d(in_channels=256,
-                                                 out_channels=256,
+        self.res_blocks = [nn.Sequential(nn.Conv2d(in_channels=self.num_filters,
+                                                 out_channels=self.num_filters,
                                                  kernel_size=3,
                                                  padding=1,
                                                  bias=False), 
-                                       nn.BatchNorm2d(256), 
-                                       nn.ReLU(inplace=True), 
-                                       nn.Conv2d(in_channels=256,
-                                                 out_channels=256,
+                                       nn.BatchNorm2d(self.num_filters), 
```

Comment:
Can we pls make this align with `nn.Conv2d` above (and same for subsequent lines)

Reply:
This whole block in general got slightly misaligned